Author: Roland Rosenfeld <roland@debian.org>
Description: Listen on localhost:8118 instead of 127.0.0.1:8118, because
 this is independent from localhost IP (127.0.0.1 vs. 127.0.1.1)
 and also supports IPv6 (Closes: #512888)
Bug-Debian: http://bugs.debian.org/512888

--- a/config
+++ b/config
@@ -780,7 +780,7 @@ logfile logfile
 #
 #        listen-address [::1]:8118
 #
-listen-address  127.0.0.1:8118
+listen-address  localhost:8118
 #
 #  4.2. toggle
 #  ============
